### PR TITLE
Include service file for 'fabric8-toggles-service'

### PR DESCRIPTION
**Pull request by xcoulon**
Signed-off-by: Xavier Coulon <xcoulon@redhat.com>

### CODE REVIEW

File: dsaas-services/f8-toggles-service.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+services:
+- hash: 43598e780bf29b9e330d48a445cce838f6523fe3
```
Comment (aditya-konarde) [inline dsaas-services/f8-toggles-service.yaml]: Change this hash to 7f988ff0c74ccbfb0cd1e56d04f698d0ff55be4d to get the IMAGE_TAG fix

Comment (xcoulon) [reply dsaas-services/f8-toggles-service.yaml]: right! let me do that right now

File: dsaas-services/f8-toggles-service.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+services:
+- hash: 7f988ff0c74ccbfb0cd1e56d04f698d0ff55be4d
+  name: fabric8-toggles
```
Comment (vpavlin) [inline dsaas-services/f8-toggles-service.yaml]: name has to be fabric8-toggles-service (matching the repo name for convenience in CI/CD)

Comment (xcoulon) [reply dsaas-services/f8-toggles-service.yaml]: oh, I missed that O_O 
thanks, @vpavlin !

**Comment by aditya-konarde** [comment]
Ah, the naming is super confusing :(

**Comment by xcoulon** [comment]
[test]

**Pull request merged**